distribute/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# makepkg is available in bullseye's backports (and newer distros) only
+# checkinstall is available in bullseye only, too
+FROM debian:bullseye-backports
+
+SHELL ["bash", "-x", "-c"]
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# note: don't build with OpenJDK > 16, as this is the runtime we ship
+RUN apt-get update && \
+    apt-get install -y openjdk-11-jdk git wget shellcheck bash make maven python3-minimal python3-pip \
+        nsis zip libfuse2 checkinstall makepkg && \
+    python3 -m pip install git+https://github.com/TheAssassin/appimagecraft.git#egg=appimagecraft
+
+# note: makepkg is contained in universe only, so we need to add that repository
```

Comment:
I don't understand what this comment is supposed to mean here. No "universe" repository is added.

Reply:
Left over from the times I used Ubuntu as a base.